src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -5091,26 +5217,40 @@ subroutine atm_compute_dyn_tend_work(nCells, nEdges, nVertices, nVertLevels_dumm
 
       ! accumulate horizontal mass-flux
 
+      !$acc parallel default(present)
+      !$acc loop gang worker
       do iCell=cellStart,cellEnd
-         h_divergence(1:nVertLevels,iCell) = 0.0
+
```

Comment:
Not sure if the newline here is required, but the style seems inconsistent from the loop on line 5178-5186